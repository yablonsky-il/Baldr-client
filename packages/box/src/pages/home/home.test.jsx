import Home from './home';

describe('<HomeUI />', () => {
  const wrapper = shallow(<Home />);

  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
