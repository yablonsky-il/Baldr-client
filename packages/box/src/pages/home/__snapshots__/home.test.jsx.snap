// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeUI /> should match snapshot 1`] = `
<div>
  <div
    style={
      Object {
        "width": "250px",
      }
    }
  >
    <form
      className="d-flex flex-column"
    >
      <WithStyles(ForwardRef(TextField))
        onChange={[Function]}
      />
      <WithStyles(ForwardRef(Button))
        className="mt-2"
        onClick={[Function]}
        variant="contained"
      >
        Send
      </WithStyles(ForwardRef(Button))>
    </form>
  </div>
</div>
`;
